libraries/helpers.rb

```diff
@@ -52,556 +52,572 @@ def kibana_artifact_checksum(version, install_method)
     end
 
     def kibana_arch
-      (node['kernel']['machine'] == 'x86_64') ? 'x86_64' : 'x86'
+      node['kernel']['machine'] == 'x86_64' ? 'x86_64' : 'x86'
     end
 
     # Returns SHA256 checksum map for Kibana artifacts
     def kibana_checksum_map
-      { '6.4.1' => {
+      { '6.4.2' => {
+        'darwin' => {
+          'x86_64' => {
+            'checksum' => '1078c4c5ced83459726522a268510e1c99d5e5f4fef7284c3b39ba8ca9c59bf2'
+          }
+        },
         'release' => {
           'x86_64' => {
-            'checksum' => '6c6fa47450cb05b96b97166a627d07c76591909da82d3bca0708bdebb889bf86',
-          },
+            'checksum' => 'f3a69709171ea8e28da754578ec519f33c423d3e3cbed6e81381f14cb8517383'
```

Comment:
why are you doing this?  this contradicts style guide and checks have failed in CI. please, don't do this

Reply:
@anuriq I ran rubocop before uploading this. Will remove shortly